doc/NAC/PnacSonicHLD.md

```diff
@@ -0,0 +1,374 @@
+                    "mode":"sr_tcm",                    
+                    "cir":"600",                    
+                    "cbs":"600",                    
+                    "red_action":"drop"                    
+            },
+            "queue2_group1": {            
+                    "cbs": "1000",                    
+                    "cir": "1000",                    
+                    "genetlink_mcgrp_name": "packets",                    
+                    "genetlink_name": "psample",                    
+                    "meter_type": "packets",                    
+                    "mode": "sr_tcm",                    
+                    "queue": "2",                    
+                    "red_action": "drop",                    
+                    "trap_action": "trap",                    
+                    "trap_priority": "1"                     
+             }             
+    },    
+    "COPP_TRAP": {    
+            "nac": {            
+                    "trap_ids": "eapol",                    
+                    "trap_group": "queue4_group3"                    
+            }            
+    }    
+}
+```
+After configurating “***NAC Adminstate***” to “***up***” for a given interface, update in config_db triggers 1following actions
+Update learn_mode for the specific interface in config_db to “***SAI_BRIDGE_PORT_FDB_LEARNING_MODE_DROP***”
+Flush FDB for the bridge that’s part of the interface
+Enable NAC for the interface maintained by hostapd. This point onwards hostapd starts listening to EAPOL packets received on this interface.
+
+After configurating “***NAC Adminstate***” to “down” for a given interface, update in config_db triggers following actions,
+Action to disable NAC for the interface maintained by hostapd. This point onwards hostapd stops listening to EAPOL packets received on this interface.
+Update learn_mode for the specific interface in config_db to “***SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW***”
+### 3.2.4 Actions performed after receiving state change notification from hostapd
+Hostapd_wrapper gets notification from hostapd whenever interface authorisation state changes from “Authorised” to “Unauthorised” and from “Unauthorised” to “Authorised”
+
+After receiving interface state change notification from Unauthorised to Authorised hostapd_wrapper performs following two operations,
+- Update column “***nac_status***” in NAC_SESSION table which is part of config_DB to "***authorised***"
+- Update learn_mode for the specific interface in config_db to “***SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW***”  
+This action will trigger Orchagent invoking SAI to update the learn_mode.
+After receiving interface state change notification from Authorised to Unauthorised hostapd_wrapper performs following two operations,
+- Update “***nac_status***” in NAC_SESSION table which is part of config_DB to "***unauthorised***".
```

Comment:
Why is this table a part of CONFIG_DB? This seems like an operational change again w.r.t PNAC.

Reply:
Acknowledged. We will update this topic.